src/callbacks.jl

```diff
@@ -76,15 +76,15 @@ function MOI.submit(model::Model, cb::MOI.UserCut, con::ScalarConstraint)
 end
 
 function MOI.submit(
-    model::Model,
+    model::GenericModel{T},
     cb::MOI.HeuristicSolution,
     variables::Vector{VariableRef},
     values::Vector{<:Real},
-)
+) where {T}
```

Comment:
So we might have to back up here and have a bigger discussion. I don't think we ever said that that variable values were something other than `Float64`. If the coefficients are `Complex`, that doesn't mean the variables are?

Reply:
`T` is still a subtype of `Real`. It's typically either `Float64` or `Rational{BigInt}`. The value of the variables have type `T` and the value of the coefficients can either have type `T` or `Complex{T}`.